snpseq_metadata/models/converter.py

```diff
@@ -62,7 +81,17 @@ def ngi_to_sra(
     def lims_to_ngi(
         cls: Type[T], lims_model: lims_model_class
     ) -> Optional[ngi_model_class]:
-        # only recurse if this is called in the base class
+        """
+        Entry point to convert a LIMS model class to a corresponding NGI model class. The method
+        will iterate over subclasses to locate a subclass whose lims_model_class matches the
+        supplied lims_model.
```

Comment:
I wonder about Lims models contains the class sequencing_container, which have no counterpart in ngi models.